environs/imagedownloads/doc.go

```diff
@@ -0,0 +1,6 @@
+// Package imagedownloads implements image-downloads metadata from
+// simplestreams.
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package imagedownloads
```

Comment:
I believe something went wrong here as you want the docs below and the Copyright above?